<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wochenansicht</class>
 <widget class="QWidget" name="Wochenansicht">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 248, 248);</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>561</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="neuer_Termin">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>805</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(200, 24, 1);
}
</string>
   </property>
   <property name="text">
    <string>Termin hinzufügen</string>
   </property>
  </widget>
  <widget class="QToolButton" name="dayBackButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(200, 24, 1);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DateLable">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>300</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 24, 1);
color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Date Label&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="faecher">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>805</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(200, 24, 1);

}</string>
   </property>
   <property name="text">
    <string>Studienfächer</string>
   </property>
  </widget>
  <widget class="QToolButton" name="dayFrontButton">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(200, 24, 1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Lernplan">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>805</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(200, 24, 1);

}</string>
   </property>
   <property name="text">
    <string>Lernplan </string>
   </property>
  </widget>
  <widget class="QPushButton" name="profil">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(200, 24, 1);

}</string>
   </property>
   <property name="text">
    <string>Profil</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>440</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="Anzeige">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>315</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QTableView
{
  border: 2px solid rgb(200, 24, 1);
  border-radius: 10px;
  padding: 0 8px;
color: rgb(0,0,0);
background-color: rgb(255, 255, 255);

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(200, 24, 1);</string>
   </property>
   <property name="text">
    <string>Das steht heute an:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Mo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>360</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Di&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Mi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Do&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Fr&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>360</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#00aa00;&quot;&gt;Sa&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>360</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#ff0000;&quot;&gt;So&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Sa">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="So">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Mo">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Di">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Mi">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Do">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Fr">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>420</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl4">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl6">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl5">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl7">
   <property name="geometry">
    <rect>
     <x>1680</x>
     <y>0</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>1921</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(6, 6, 6);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="ansicht">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>200</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QComboBox
{
 border: 2px solid gray;
  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(200, 24, 1);

}</string>
   </property>
   <item>
    <property name="text">
     <string>Wochenansicht</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tagesansicht</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monatsansicht</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>290</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 24, 1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LogoKlein">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>940</y>
     <width>248</width>
     <height>140</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcomelbl2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(200, 24, 1);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Welcomelbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(200, 24, 1);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Abmelden">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>990</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

  border-radius: 10px;
  padding: 0 8px;
color: rgb(255, 255, 255);
background-color: rgb(200, 24, 1);

}</string>
   </property>
   <property name="text">
    <string>Abmelden</string>
   </property>
  </widget>
  <widget class="QLabel" name="blackBackround">
   <property name="geometry">
    <rect>
     <x>1799</x>
     <y>89</y>
     <width>66</width>
     <height>66</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(171, 171, 171);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="profilePicture">
   <property name="geometry">
    <rect>
     <x>1800</x>
     <y>90</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>491</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QTableView
{
  border: 2px solid rgb(200, 24, 1);
  border-radius: 10px;
  padding: 0 8px;
	background-color: rgb(255, 255, 255);
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="terminlabel">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>275</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(200, 24, 1);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>1870</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 22, 25);

color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <zorder>label_12</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>tableView</zorder>
  <zorder>neuer_Termin</zorder>
  <zorder>dayBackButton</zorder>
  <zorder>DateLable</zorder>
  <zorder>faecher</zorder>
  <zorder>dayFrontButton</zorder>
  <zorder>Lernplan</zorder>
  <zorder>profil</zorder>
  <zorder>Anzeige</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>Sa</zorder>
  <zorder>So</zorder>
  <zorder>Mo</zorder>
  <zorder>Di</zorder>
  <zorder>Mi</zorder>
  <zorder>Do</zorder>
  <zorder>Fr</zorder>
  <zorder>lbl2</zorder>
  <zorder>lbl4</zorder>
  <zorder>lbl1</zorder>
  <zorder>lbl3</zorder>
  <zorder>lbl6</zorder>
  <zorder>lbl5</zorder>
  <zorder>lbl7</zorder>
  <zorder>line</zorder>
  <zorder>ansicht</zorder>
  <zorder>LogoKlein</zorder>
  <zorder>Welcomelbl2</zorder>
  <zorder>Welcomelbl</zorder>
  <zorder>Abmelden</zorder>
  <zorder>blackBackround</zorder>
  <zorder>profilePicture</zorder>
  <zorder>tableView_2</zorder>
  <zorder>terminlabel</zorder>
  <zorder>exit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
